src/pages/auth/Forgot.jsx

```diff
@@ -48,7 +48,7 @@ export default function Forgot() {
     <SimpleFormPage
```

Comment:
Why this SimpleFormPage component twice?

Reply:
There are sort of two different pages in this component! 
1. forgot password? give us your email
2. email sent!